app/views/articles/_liquid.html.erb

```diff
@@ -14,7 +14,7 @@
   <a href='<%= article.path %>' class='ltag__link__link'>
     <div class='ltag__link__content'>
       <h2><%= title %></h2>
-      <h3><%= article.user.name %> ・ <%= article.reading_time %> min read</h3>
+      <h3><%= article.user.name %> ・ <%= article.published_at %> ・ <%= article.reading_time %> min read</h3>
```

Comment:
Hi @12vanblart, I think you should use `article.readable_publish_date` for consistency

Reply:
That sounds like a more appropriate option - Updated!